packages/manager/apps/web/client/app/hosting/email/activate/activate.routing.js

```diff
@@ -0,0 +1,30 @@
+import endsWith from 'lodash/endsWith';
+import filter from 'lodash/filter';
+import sortBy from 'lodash/sortBy';
+import union from 'lodash/union';
+
+import {
+  WEB_HOSTING_NOT_USABLE_DOMAINS,
+} from './activate.constants';
+
+export default /* @ngInject */($stateProvider) => {
+  $stateProvider.state('app.hosting.activate', {
+    url: '/activate',
+    component: 'webHostingEmailActivateComponent',
+    resolve: {
+      serviceName: /* @ngInject */ $transition$ => $transition$.params().productId,
```

Comment:
This isn't already resolve in the parent ? Imo it makes more sense to declare it when the `productId` param is declared in the route